deps/patches/mbedtls-ssl.h.patch

```diff
@@ -0,0 +1,12 @@
+--- ssl.h.old	2016-06-28 18:12:06.000000000 +0530
++++ ssl.h	2016-08-03 18:51:34.000000000 +0530
+@@ -54,7 +54,8 @@
+ #endif
+ 
+ #if defined(MBEDTLS_HAVE_TIME)
+-#include <time.h>
++//#include <time.h>
++#include "platform.h"
```

Comment:
This is a known bug fixed upstream, and hopefully should be in mbedtls 2.3.1. Maybe mac only, but the fix should work everywhere (I think).

https://github.com/ARMmbed/mbedtls/issues/522